### PR TITLE
Update Search flow and test out with search with audio details

### PR DESCRIPTION
**Pull request by raigex**

### CODE REVIEW

File: search/src/main/res/layout/fragment_search_audio_detail.xml [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+<android.support.design.widget.CoordinatorLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="@color/audioBackground"
+    android:fitsSystemWindows="true"
+    app:statusBarBackground="@android:color/transparent">
+
+    <android.support.design.widget.AppBarLayout
```
Comment (Cliabhach) [inline search/src/main/res/layout/fragment_search_audio_detail.xml]: If this is the same as one of the other layouts' AppBarLayouts, then we should probably pull that out into an `include`.

File: search/src/main/res/layout/search_app_bar_layout.xml [hunk 2]
```diff
@@ -3,7 +3,8 @@
     android:layout_width="match_parent"
     android:layout_height="wrap_content"
     android:background="@color/greyText"
-    android:elevation="5dp">
+    android:elevation="5dp"
+    android:fitsSystemWindows="true">
```
Comment (Cliabhach) [inline search/src/main/res/layout/search_app_bar_layout.xml]: I guess we needed this?

File: search/src/main/res/navigation/search_sub_navigation_graph.xml [hunk 3]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<navigation xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/search_sub_navigation_graph.xml"
```
Comment (Cliabhach) [inline search/src/main/res/navigation/search_sub_navigation_graph.xml]: Remove `.xml`

File: ui/src/main/kotlin/edu/artic/ui/BaseFragment.kt [hunk 4]
```diff
@@ -104,9 +110,14 @@ abstract class BaseFragment : Fragment() {
                 requireActivity().setWindowFlag(WindowManager.LayoutParams.FLAG_TRANSLUCENT_STATUS, false)
                 requireActivity().window?.statusBarColor = Color.TRANSPARENT
             } else {
-                val primaryDarkColor = intArrayOf(android.support.design.R.attr.colorPrimaryDark)
-                requireContext().getThemeColors(primaryDarkColor).getOrNull(0)?.defaultColor?.let {
-                    requireActivity().window?.statusBarColor = it
+                Timber.d("updateToolbar: customToolbarColorResource $customToolbarColorResource")
```
Comment (Cliabhach) [inline ui/src/main/kotlin/edu/artic/ui/BaseFragment.kt]: Auto-indent, please.

File: search/src/main/res/navigation/search_sub_navigation_graph.xml (hunk 3 above)
Comment (raigex) [reply search/src/main/res/navigation/search_sub_navigation_graph.xml]: done

File: search/src/main/res/layout/fragment_search_audio_detail.xml (hunk 1 above)
Comment (raigex) [reply search/src/main/res/layout/fragment_search_audio_detail.xml]: Yes this can be done in a different merge. Will create a chore ticket for it.

**Pull request merged**